contracts/KlerosPOC.sol

```diff
@@ -309,13 +309,14 @@ contract KlerosPOC is Arbitrator {
 
         uint winningChoice=dispute.voteCounter[dispute.appeals].winningChoice;
         uint amountShift=(alpha*minActivatedToken)/ALPHA_DIVISOR;
+        dispute.appealsRepartitioned.length++; // Make new AppealsRepartitioned
         for (uint i=0;i<=dispute.appeals;++i) {
             // If the result is not a tie, some parties are incoherent. Note that 0 (refuse to arbitrate) winning is not a tie.
             // Result is a tie if the winningChoice is 0 (refuse to arbitrate) and the choice 0 is not the most voted choice.
             // Note that in case of a "tie" among some choices including 0, parties who did not vote 0 are considered incoherent.
             if (winningChoice!=0 || (dispute.voteCounter[dispute.appeals].voteCount[0] == dispute.voteCounter[dispute.appeals].winningCount)) {
-                uint totalToRedistibute=0;
```

Comment:
In one shot, no need to save that. It’s cheaper to modify variables in memory than those in storage.